docs/7.x/requirements.md

```diff
@@ -7,21 +7,17 @@ Kubernetes and Gravity.
 
 Gravity supports the following Linux distributions:
 
-| Linux Distribution        | Version         | Docker Storage Drivers                |
+| Linux Distribution       | Version         | Docker Storage Drivers                |
 |--------------------------|-----------------|---------------------------------------|
-| Red Hat Enterprise Linux | 7.2-7.3         | `devicemapper`*                        |
-| Red Hat Enterprise Linux | 7.4-7.7, 8.0    | `devicemapper`*, `overlay`, `overlay2` |
-| CentOS                   | 7.2-7.7         | `devicemapper`*, `overlay`, `overlay2` |
-| Debian                   | 8-9             | `devicemapper`*, `overlay`, `overlay2` |
-| Ubuntu                   | 16.04, 18.04    | `devicemapper`*, `overlay`, `overlay2` |
-| Ubuntu-Core              | 16.04           | `devicemapper`*, `overlay`, `overlay2` |
+| Red Hat Enterprise Linux | 7.4-7.7, 8.0    | `overlay`, `overlay2`                 |
```

Comment:
ha, this is pretty handy table. We've customers on RHEL5 and CentOS6 using Teleport, a chart like this would be helpful for our customers to let them know which version work with which version of Teleport.